test/common/tls/integration/ssl_integration_test.cc

```diff
@@ -275,6 +275,40 @@ TEST_P(SslIntegrationTest, RouterDownstreamDisconnectBeforeResponseComplete) {
   checkStats();
 }
 
+// Test server preference of cipher suites. Default server order is ECDHE-RSA-AES128-GCM-SHA256
```

Comment:
In these two tests, it will be clearer what is happening, and less likely to break in the future, if you explicitly set the server ciphers as well, instead of relying on the default.